src/metabase/api/permission_graph.clj

```diff
@@ -75,7 +75,9 @@
 
 (s/def ::data-model (s/keys :opt-un [::native ::schemas]))
 
-(s/def ::db-perms (s/keys :opt-un [::data ::download ::data-model]))
+(s/def ::details (s/or :str->kw #{"yes" "no"}))
```

Comment:
Specifically, I worry that there might be a misconstrual of the thing to think that a {:detail :no} post-parse of permission was a thing, despite the actual semantics as far as I understand it being clearing the thing (as in the test)

Reply:
I'm not totally clear on what you mean by the last comment, what does "the actual semantics as far as I understand it being clearing the thing" mean?

The main reasons for `:yes` and `:no` keywords instead of bools is basically consistency with the language used on the frontend, and consistency with general perms which also use `:yes` and `:no. And the rest of the graph uses keywords so it would be weird to have bools here.

I'll put a note that general perms also use `yes` and `no` but I'm not sure if that's exactly what you're commenting on